src/index.ts

```diff
@@ -219,6 +219,7 @@ const createWindow = () => {
     titleBarStyle: isMac ? 'hidden' : 'default',
     frame: isLinux,
     backgroundColor,
+    icon: join(__dirname, 'build-helpers/images/icons/16x16.png'),
```

Comment:
Can you provide a screenshot showing the difference between with icon and without?

According to the electron documentation it supposedly is optional: "On Windows it is recommended to use ICO icons to get best visual effects, you can also leave it undefined so the executable's icon will be used."

See https://www.electronjs.org/docs/latest/api/browser-window#new-browserwindowoptions > "icon"